### DISCUSSION TITLE
ECMO in MIMIC

### DISCUSSION THREAD

**Discussion by aegis301** [MIT-LCP/mimic-code#1713] [MIMIC-IV]
Dear Mimic-Community,

i'm interested in working with MIMIC on a project concerning patients that received ECMO therapy during their stay. I want to first identify patients what were hooked up to an ECMO during their stay. I wrot the following query to check for item ids concerned with ECMO therapy:
```sql
SELECT
      stay_id,
      MIN(charttime) AS ecmo_start,
      MAX(charttime) AS ecmo_end
    FROM
      mimiciv_icu.chartevents
    WHERE
      itemid = 229268 -- circuit configuration
      AND value = 'VA' -- filter for VA ECMO
    GROUP BY
      stay_id
```
I want to explicitely filter for patients that received veno-arterial ECMO therapy. By doing this I get back 73 patients that received this kind of therapy. Seems to be a very low number. Does anyone have experience with this patient collective in MIMIC and has an explanation for this? Or am I missing something?

Kind regards

**Comment by rbaccile** [comment]
Hi, also wondering about this. Additionally, does anyone know what (CH) might mean with respect to ECMO? There are items for flow, speed, sweep, and FiO2 that are labeled (ECMO) and (CH).
